### PR TITLE
rename name to displayName

### PR DESCRIPTION
**Pull request by petersg83**

### CODE REVIEW

File: packages/core/content-type-builder/server/services/components.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const formatComponent = component => {
     apiId: modelName,
     schema: {
       icon: _.get(info, 'icon'),
-      name: _.get(info, 'name') || _.upperFirst(pluralize(uid)),
+      displayName: _.get(info, 'displayName') || _.upperFirst(pluralize(uid)),
```
Comment (alexandrebodin) [inline packages/core/content-type-builder/server/services/components.js:23]: if displayName is required can we remove the second operand of the OR ? removing pluralize call as much as possible

**Review by alexandrebodin — APPROVED** [review]
One comment but looks good to me.

**Comment by derrickmehaffy** [comment]
Is this a change from previous betas or just a fix (asking in case we need to tell Piwi that documentation needs to change)

**Comment by petersg83** [comment]
@derrickmehaffy It's a change, to be updated in the documentation yes. I put it in notion in "Documentation Changes"

**Review by alexandrebodin — APPROVED** [review]
LGTM waiting for the tests

**Pull request merged**